### DISCUSSION TITLE
Thanks Streetmerchant, I got a RTX 3080

### DISCUSSION THREAD

**Discussion by jiglesiasc** [jef/streetmerchant#971] [General]
I just wanted to thank to Jef and Streetmerchant I would have never made it without the bot!!!

I got the Zotac AMP Holo RTX 3080. Attached proof of successful order.

![imagen](https://user-images.githubusercontent.com/62509598/100434199-e3261800-307a-11eb-9e76-7291a88f1109.png)

A few advises:

- Enable sound notification and the auto launch browser, you gotta be fast, really fast.
- Have your PayPal account already logged in, ready to shot
- Run separate instances of streetmerchant as this way you can scan faster
- Customize your streetmerchant, I added bundles from Newegg and a few links that hasn't been yet added to the main version of streetmerchant

**Comment by controledxchaos** [comment]
I cant figure out how to add bundles. also my bot has been running and i never got a 3080 notification. is there a place that has instructions on how to add links?

**Comment by jiglesiasc** [comment]
> 
> 
> I cant figure out how to add bundles. also my bot has been running and i never got a 3080 notification. is there a place that has instructions on how to add links?

There was a big drop of a few hours ago in Amazon for the EVGA XC3 3080, like 100 units. I definitely recommend running at least 4 separate instances of Streetmerchant. one for Newegg, other for Newegg bundles, other for Amazon, other for Zotac...

The issue with having a lot of stores all of them in just one instance is that it takes too long for the script to go back and check a link, it might takes minutes and that's a huge amount of time with the current demand.

About your question regarding how to add bundles, I don't think there is an instruction on how to do that. What I did was next:

- Let's say you want to add the following bundle: https://www.newegg.com/Product/ComboDealDetails?ItemList=Combo.4191565
- First you need to add a key name to the file: streetmerchant\src\store\model\store.ts in the section called "export type Model =" Just follow the same structure of the already existing entries. You could use something like 'eagle psu bundle'.
- The you need to add the following to the file: \streetmerchant\src\store\model\newegg.ts ... 
`{
			brand: 'gigabyte',
			cartUrl:
				'https://secure.newegg.com/Shopping/AddtoCart.aspx?Submit=ADD&ItemList=Combo.4191565',
			itemNumber: '14-932-336',
			model: 'eagle psu bundle',
			series: '3080',
			url:
				'https://www.newegg.com/Product/ComboDealDetails?ItemList=Combo.4191565'
		},`

To be honest the itemNumber there is wrong, but I have no idea how that affects since I don't use the auto add to the cart option.

And that should work. Actually the AMP Holo that I got wasn't part of the original list of supported models I customized the files to include that model.

**Comment by controledxchaos** [comment]
So i have to install streetmerchant in separate folders to run separate instances?

**Comment by jiglesiasc** [comment]
> 
> 
> So i have to install streetmerchant in separate folders to run separate instances?

I just copy and paste the whole folder. It will run just fine

**Comment by controledxchaos** [comment]
Awesome thanks!
last question...have you been running a VPN?

**Comment by jiglesiasc** [comment]
No I haven't ran streetmerchant over a VPN. Is really important that your streetmerchant scans the same link every 15 seconds max, otherwise you have no chance against scalper's bots

**Comment by jef** [comment]
Awesome!!! Congratulations. I hope you enjoy it! 😄
